cmd/keytransparency-sequencer/main.go

```diff
@@ -228,4 +228,13 @@ func runSequencer(ctx context.Context, conn *grpc.ClientConn,
 			glog.Errorf("PeriodicallyRun(RunBatchForAllMasterships): %v", err)
 		}
 	})
+
+	sequencer.PeriodicallyRun(ctx, time.Tick(*refresh), func(ctx context.Context) {
+		if err := signer.AddAllDirectories(ctx); err != nil {
```

Comment:
I think you have a data race here, since both periodic jobs are trying to add directories.